src/core/qgis.h

```diff
@@ -91,6 +91,58 @@ class CORE_EXPORT Qgis
     // Enumerations
     //
 
+    /**
+     * Authentication configuration storage capabilities.
+     *
+     * \since QGIS 3.40
+     */
+    enum class AuthConfigurationStorageCapability SIP_MONKEYPATCH_SCOPEENUM_UNNEST( QgsAuthConfigurationStorage, Capability ): int SIP_ENUM_BASETYPE( IntFlag )
+    {
+      NoCapabilities = 0, //!< Item has no capabilities
```

Comment:
```suggestion
```

This shouldn't be necessary, since a default constructed AuthConfigurationStorageCapabilities() flags object will implicitly have no capabilities